### PR TITLE
PARQUET-2198 : Updating jackson data bind version to fix CVEs

### PR DESCRIPTION
**Pull request by avinashkolluru**
Fixes  CVE-2022-42003 and  CVE-2022-42004

Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Parquet Jira](https://issues.apache.org/jira/browse/PARQUET/) issues and references them in the PR title. For example, "PARQUET-2198: My Parquet PR"
  - https://issues.apache.org/jira/browse/PARQUET-2198

### CODE REVIEW

**Comment by tooptoop4** [comment]
🥇

File: pom.xml [hunk 1]
```diff
@@ -74,7 +74,7 @@
     <jackson.datatype.groupId>com.fasterxml.jackson.datatype</jackson.datatype.groupId>
     <jackson.package>com.fasterxml.jackson</jackson.package>
     <jackson.version>2.13.2</jackson.version>
```
Comment (frant-hartm) [inline pom.xml:76]: The jackson version should be updated as well so the versions match.
```suggestion
    <jackson.version>2.13.4</jackson.version>
```

**Review by frant-hartm — CHANGES_REQUESTED** [review]

**Review by frant-hartm — APPROVED** [review]
Looks ok to me now. However I am not a parquet committer.

**Comment by shangxinli** [comment]
LGTM

**Comment by mr1716** [comment]
Hey, whats the timetable for merging this @shangxinli @frant-hartm ?

**Comment by satish-mittal** [comment]
@shangxinli can we release a new version that fixes these two vulnerabilities?

**Comment by shangxinli** [comment]
Yeah, we will release soon

**Comment by botchniaque** [comment]
I don't have now any specifics, but I guess there were some differences in versions `2.13.x` and `2.14.x` and the scala versions compatibility. This may be the cause why `2.13.x` is used. But you're right - there are newer versions available eg. https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-databind/2.13.5

**Comment by steveloughran** [comment]
+been some issues with transient dependencies from jackson releases in hadoop, hence "HADOOP-18332. Remove rs-api dependency by downgrading jackson to 2.12.7.". jersey 1.0 coexistence.

**Comment by vimal3271** [comment]
Could you help in releasing the new version which fixes the CVEs ?

**Comment by shangxinli** [comment]
We will release it soon

**Comment by mdadil-dk** [comment]
Any new release plan for this ?? Or have SNAPSHOT/RC build to test ??

**Comment by shangxinli** [comment]
We already started working on the release. Please wait...

**Comment by steveloughran** [comment]
@mdadil-dk there's a source code repository there for you to checkout and build your own -SNAPSHOT artifacts. Doing local builds and testing against your own code, even before RCs are up- is the best way to identify integration problems with your apps *and get them fixed*.

**Comment by nikhilenr** [comment]
Hi All,

New parquet-jackson version is released and reported cves are resolved with v 1.13.0.
https://mvnrepository.com/artifact/org.apache.parquet/parquet-jackson/1.13.0

Thanks @shangxinli for fixing.

**Pull request merged**